cookbook/setup_data/customize_dataset.rst

```diff
@@ -0,0 +1,104 @@
+How to Customize the Dataset
+============================
+
+Akeneo PIM ships with two datasets. They are located in `vendor/akeneo/pim-community-dev/src/Pim/Bundle/InstallerBundle/Resources/fixtures`:
+* `icecat-demo-dev` to be able to take a look and play with the PIM with already preset families, categories, products, etc..
+* `minimal` to start a brand new project
```

Comment:
maybe: "to start a blank PIM project" ? I would have made appear the word blank somewhere.